lib/ramble/ramble/test/modifier_functionality/mock_env_var_modifiers.py

```diff
@@ -10,6 +10,7 @@
 
 import pytest
 
+from ramble.test.dry_run_helpers import *
```

Comment:
It's not a big deal but why do you `from .. import *` in here not just `import module` ?

Reply:
Laziness mostly. 😏 I should at a minimum make it `from ... import <stuff>` instead of `*`.

Reply:
Yea that's what caught my eye, I think that's generally considered "better" (or just import the entire module)